src/cbapi/psc/threathunter/query.py

```diff
@@ -285,28 +303,31 @@ def _validate(self, args):
     def _search(self, start=0, rows=0):
         # iterate over total result set, 100 at a time
         args = self._get_query_parameters()
-        self._validate(args)
+        #self._validate(args)
```

Comment:
You can use the v1 validation endpoint for this, I believe the `_validate` function just needs to pull the query from `query` instead of `q`.

Reply:
You're correct, I've updated the `_validate()` function in the PR by grabbig the `query` parameter from `args` and submitting it as the value for `q` to the function. Validation now passes as expected.